bin/verilator

```diff
@@ -3405,6 +3405,42 @@ behavior.  See the test_regress/t/t_dpi_display.v file for an example.
 Same as C<sformat> in configuration files, see L</"CONFIGURATION FILES">
 for more information.
 
+=item /*verilator split_var*/
+
+Attached to a variable or a net declaration. It breaks the variable into
+multiple pieces. The following data types are supported.
+
+  Packed array and packed struct consist of Integer types (reg, logic, bit, byte, int...)
+  unpacked array of data type above
```

Comment:
Attached to a variable or a net declaration to break the variable into
multiple pieces to work around UNOPTFLAT and other optimization issues.
Typically the UNOPTFLAT warning will suggest which variables this might be
good to attach to.

Supports arrays and packed structs with integer types.

As adding split_var creates additional internal state, this may slow down
the Verilation and C compiler runtimes if used unnecessarily.